[RFC] TSC Governance Proposal: Document the TSC Meetings host rotation

arm4b opened the pull request:
Proposal to the Governance describing the TSC Meeting Host rotation.
This would document the processes we already had for the past several months when the meeting host responsibility was rotated between the TSC members.

Closes https://github.com/StackStorm/st2/pull/5135, Closes https://github.com/StackStorm/st2/pull/5071

### Other discussions
Previous discussions: [Rotation for the Monthly TSC Meeting Manager/Coordinator #4](https://github.com/StackStorm/community/issues/4)
TSC Meeting procedure & host responsibilities: [StackStorm TSC Meetings #33](https://github.com/StackStorm/community/issues/33)

### Current status
In consultation, waiting for review/feedback from @StackStorm/tsc

Commit:
TSC Governance: Document TSC Meetings rotation

Commit:
Add more clarification for the TSC meetings & host priority

amanda11 commented on GOVERNANCE.md:
```diff
@@ -9,6 +9,16 @@ It extends [Technical Charter](https://stackstorm.com/wp/wp-content/uploads/2020
+
+* TSC Meetings Host responsibility is rotated between the TSC maintainers.
+* Priority to host is given based on activity: Current Release Owners > Maintainers who weren't in rotation > Other Maintainer Volunteers.
+* Maintainers are free to decide collectively who would run the next TSC meeting. They can run in pairs, swap, however for diversification reasons no Maintainer can be a host for more than one in three consecutive meetings.
```
I wasn't sure if this was a bit restrictive saying you can't be the host once for every 3 meetings, perhaps a better restriction is you can't be the same host for three or more consecutive meetings. If you are a release manager you might be the host for a couple of meetings if its nearing release time, and I think last time we had a meeting the volunteer for the host was the person who did it.
So maybe allowing 2 but not 3 consecutive would be a better restriction?

armab replied:
Thanks :+1: was looking for feedback on this

Removed the limitation. Just stating the TSC Meeting rotation should be sufficient, hope everything else like the mechanism will follow naturally.

Commit:
Remove host limitation when rotating the TSC meetings

Just state in the governance the intent of TSC Meetings rotation. The mechanism will follow naturally.

cognifloyd commented on GOVERNANCE.md:
```diff
@@ -9,6 +9,16 @@ It extends [Technical Charter](https://stackstorm.com/wp/wp-content/uploads/2020
 
+### TSC Meetings
+Because of the different schedules and time zones, TSC is keen to prioritize asynchronous collaboration and work.
+However, every `1st Tuesday` of the month `09:30 AM US Pacific`, [@StackStorm/tsc](OWNERS.md) members are gathering for `1 hour` to discuss the project problems, roadmap, priorities to help the StackStorm as a project evolve and succeed.
```
Let's change this to say that we have a TSC meeting every month. Please see https://github.com/StackStorm/community/issues/33 for the schedule.

armab replied:
Thanks, updated in https://github.com/StackStorm/st2/pull/5711/commits/534e604a0375411c262a8e0393d49eaa25751a53

Commit:
Link to the TSC Meetings page with more info about the schedule

cognifloyd commented on GOVERNANCE.md:
```diff
@@ -9,6 +9,16 @@ It extends [Technical Charter](https://stackstorm.com/wp/wp-content/uploads/2020
 
+### TSC Meetings
+Because of the different schedules and time zones, TSC is keen to prioritize asynchronous collaboration and work.
+However, every month, [@StackStorm/tsc](OWNERS.md) members are gathering for `1 hour` to discuss the project problems, roadmap, priorities to help the StackStorm as a project evolve and succeed.
```
```suggestion
However, every month, [@StackStorm/tsc](OWNERS.md) members gather for `1 hour` to discuss the project problems, roadmap, priorities to help the StackStorm as a project evolve and succeed.
```

cognifloyd reviewed (approved):
Looks good. Just one minor grammar nit.

arm4b committed:
Update GOVERNANCE.md with the minor wording fixes

Co-authored-by: Jacob Floyd <cognifloyd@gmail.com>

amanda11 reviewed (approved):
LGTM

rush-skills reviewed (approved):
LGTM

bishopbm1 reviewed (approved):
LGTM

rush-skills commented:
@armab Is this ready to merge? Or do we need more @StackStorm/tsc votes?

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/StackStorm/st2?pullRequest=5711) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/StackStorm/st2?pullRequest=5711) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: arm4b<br/>:x: Eugen Cusmaunsa<br/><hr/>**Eugen Cusmaunsa** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/StackStorm/st2?pullRequest=5711) it.</sub>

mamercad reviewed (approved):
🚢
